src/relayserver/TransactionManager.ts

```diff
@@ -226,11 +226,12 @@ data         | 0x${transaction.data.toString('hex')}
       if (shouldRecheck) {
         const receipt = await this.contractInteractor.getTransaction(transaction.txId)
         if (receipt == null) {
-          log.warn(`failed to fetch receipt for tx ${transaction.txId}`)
+          log.warn(`warning: failed to fetch receipt for tx ${transaction.txId}`)
           continue
         }
         if (receipt.blockNumber == null) {
-          throw new Error(`invalid block number in receipt ${JSON.stringify(receipt)}`)
+          log.warn(`warning: null block number in receipt for ${transaction.txId}`)
```

Comment:
And this happens a lot and I have no idea why. Not mined yet but here is your receipt? Is this how this RPC call supposed to work? Not what the official wiki says.

```
blockNumber: QUANTITY - block number where this transaction was in.
...
...
contractAddress : DATA, 20 Bytes - The contract address created, if the transaction was a contract creation, otherwise null.
```

Reply:
I didn't remove it, since I don't understand when it happens. just removed the stack trace, which is complete nonsense.